### PR TITLE
updated botathon page

### PR DESCRIPTION
**Pull request by Peyton232**
botathon page is now up to date

### CODE REVIEW

File: about/our-team.php [hunk 1]
```diff
@@ -220,10 +220,29 @@
                         </div>
                     </div>
                 </div>
+                <!-- Event Coordinator -->
+                <div class="cell-lg-12 bio-area">
+                    <div class="range range-sm-middle">
+                        <div class="cell-md-3"><img src="/images/bio-pics/abdus-samee.jpg" alt="" width="360" height="404" class="img-responsive" />
+                        </div>
+                        <!-- This is a Noel -->
```
Comment (ntindle) [inline about/our-team.php:228]: Fix this line

File: botathon/index.php [hunk 2]
```diff
@@ -223,6 +223,8 @@
 		<section class="section-50" id="field-preview">
 			<div class="shell">
 			  <h2>Field preview</h2>
+                <h3>TBA</h3>
```
Comment (ntindle) [inline botathon/index.php:226]: Coming soon instead

File: botathon/index.php [hunk 3]
```diff
@@ -259,31 +262,49 @@
...
                 	<div class="inset-md-right-30 inset-lg-right-0">
 						
-						<h4>Base Kit</h4>
-						<p>Each team gets two base kits, which consist of the below items.</p>
-						
-						<table class="parts basekit">
+						<h4>Base Kits</h4>
+						<p>Each team will get to choose which base kit they want to build their pirate ship with.</p>
+
+                        <h4>OMNI WHEEL CAR ROBOT:</h4>
```
Comment (ntindle) [inline botathon/index.php:268]: Add count of each

File: about/our-team.php (hunk 1 above)
Comment (Peyton232) [reply about/our-team.php]: removed noel line

File: botathon/index.php (hunk 2 above)
Comment (Peyton232) [reply botathon/index.php]: changed TBA

File: botathon/index.php (hunk 3 above)
Comment (Peyton232) [reply botathon/index.php:268]: added count of each robot kit

**Review by ntindle — APPROVED** [review]

**Pull request merged**